public/index.html

```diff
@@ -168,6 +168,7 @@
       flowType: 'messaging',
       localStorage: true,
       attachmentsEnabled: true,
+      showNodeUUIDs: true,
```

Comment:
Can we keep the consistency?

Looks like all the boolean keys are in singular and this one is in the plural.

Reply:
Also, can  we make it **showNodeLable** ?

Reply:
Updated